tasks/elasticsearch-config.yml

```diff
@@ -86,8 +79,8 @@
 - name: Delete Default Logging File
   file: dest=/etc/elasticsearch/logging.yml state=absent
 
-- name: Delete Default Logging File (5.x)
+- name: Delete Default Logging File
   file: dest=/etc/elasticsearch/log4j2.properties state=absent
 
-- name: Delete Default JVM Options File (5.x)
+- name: Delete Default JVM Options File
```

Comment:
Thx for information and you work!
Can you update systemd unit also ? (https://github.com/gingerwizard/ansible-elasticsearch/blob/master/templates/systemd/elasticsearch.j2#L7)